pkg/controller/podgroup.go

```diff
@@ -238,6 +238,12 @@ func (ctrl *PodGroupController) syncHandler(key string) error {
 					failed++
 				}
 			}
+		} else {
```

Comment:
It may be a little duplicate of L248-L250. 
I think it would be better to rewrite it like this, WDYT?
```go
	for _, pod := range pods {
		switch pod.Status.Phase {
		case v1.PodRunning:
			running++
		case v1.PodSucceeded:
			succeeded++
		case v1.PodFailed:
			failed++
		}
	}

       pgCopy.Status.Failed = failed
       pgCopy.Status.Succeeded = succeeded
       pgCopy.Status.Running = running

       if len(pods) == 0 {
            pgCopy.Status.Phase = schedv1alpha1.PodGroupPending
            break
       }
```

Reply:
That makes sense to me. Thanks.

Reply:
done.